packer/linux/conf/bin/bk-install-elastic-stack.sh

```diff
@@ -92,20 +90,20 @@ set_unless_present "AWS_DEFAULT_REGION" "$AWS_REGION"
 set_unless_present "AWS_REGION" "$AWS_REGION"
 EOF
 
-if [[ "${BUILDKITE_AGENT_RELEASE}" == "edge" ]] ; then
-	if [[ "$(uname -m)" == "aarch64" ]] ; then
-	  AGENT_ARCH="arm64"
-	else
-	  AGENT_ARCH="amd64"
-	fi
-	echo "Downloading buildkite-agent edge..."
-	curl -Lsf -o /usr/bin/buildkite-agent-edge \
-		"https://download.buildkite.com/agent/experimental/latest/buildkite-agent-linux-${AGENT_ARCH}"
-	chmod +x /usr/bin/buildkite-agent-edge
-	buildkite-agent-edge --version
+if [[ "${BUILDKITE_AGENT_RELEASE}" == "edge" ]]; then
+  if [[ "$(uname -m)" == "aarch64" ]]; then
+    AGENT_ARCH="arm64"
+  else
+    AGENT_ARCH="amd64"
+  fi
```

Comment:
Elsewhere you've got a nice case statement, how about doing that consistently here?

```suggestion
  case $(uname -m) in
    x86_64)    ARCH=amd64;;
    aarch64)   ARCH=arm64;;
    *)         ARCH=unknown;;
  esac
```

(although this is very nitpicky, feel free to ignore me 😅)

Reply:
There's many more examples of sprinkled throughout the code base. I can't accept this as is thought, the variable was `$AGENT_ARCH`, but the suggestion has it as `$ARCH`. It's probably nicer to standardise on $ARCH if it does not exist in this script, though.

Reply:
Ah whoops, i missed that `$AGENT_ARCH`!  Glad someone is paying attention 😅